api/apps/api/src/modules/features/features/domain/feature.ts

```diff
@@ -0,0 +1,39 @@
+import { FeatureFragment } from './feature-fragment';
+import { CustomFeature } from '@marxan-api/modules/features/features/domain/custom-feature';
+
+export enum FeatureTag {
+  Species = 'species',
+  Bioregional = 'bioregional',
+}
```

Comment:
be mindful that we already have two slightly different instances of this (I am guilty as I introduced the second one recently) - see `@marxan/features` for what I think should be the master enum